packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -983,6 +992,34 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.members;
   }
 
+  /**
+   * Reference to the mediaMetrics object
+   * @returns {MediaMetrics}
+   * @public
+   * @memberof Meeting
+   */
+  getMediaMetrics() {
```

Comment:
I know for consistency sake we are using getters around the meetings plugin, but was there a particular reason not to expose just the `this.mediaMetrics` property?

Reply:
no particular reason, just consistency sake.